crates/ruff_python_stdlib/src/identifiers.rs

```diff
@@ -1,6 +1,13 @@
+use crate::keyword::KWLIST;
+
 /// Returns `true` if a string is a valid Python identifier (e.g., variable
 /// name).
 pub fn is_identifier(s: &str) -> bool {
+    // Is the string a keyword?
+    if KWLIST.contains(&s) {
+        return false;
+    }
```

Comment:
Can we go through the usages of this method, and see if we're checking for `KWLIST` already at the call-sites? If so, can we then remove those extra checks?

Reply:
`KWLIST` is used for converting named_tuple or typed_dict.
Will remove it.